lib/Client.js

```diff
@@ -7,9 +7,22 @@ var min = require('osc-min');
 var Message = require('./Message');
 
 var Client = function (host, port) {
+    var isBroadcast = host.split('.').reduce(function(previous, current){
```

Comment:
the heuristic of doing a reduce is a bit off... theoretically if 255 occurs in the last section of the IP it could qualify as broadcast right?

Also Multicast ips wouldn't qualify such as --> 224.0.0.1